lib/galaxy/tools/deps/resolvers/conda.py

```diff
@@ -88,11 +88,14 @@ def get_option(name):
         auto_install = _string_as_bool(get_option("auto_install"))
         copy_dependencies = _string_as_bool(get_option("copy_dependencies"))
 
-        if not os.path.exists(conda_context.conda_prefix):
+        if not conda_context.is_conda_installed():
             if auto_init:
-                if install_conda(conda_context):
+                if conda_context.can_install_conda():
```

Comment:
Damnit, this causes a race condition when starting with a multi-hander setup :/.
If you have no conda installation in `conda_prefix`, all handlers will attempt to install conda at the same time (which fails because _conda/ is not empty), and so n-1 handler will have conda deactivated the first time you try to install conda at startup. Any idea what to do here, @bgruening @jmchilton ? Maybe a lockfile?